### PR TITLE
Make Node.textContent ignore comment nodes

### PR DESCRIPTION
**Pull request by blois**
Currently it is incorrectly including the text from comment nodes in generating textContent.

### CODE REVIEW

File: test/js/Node.js [hunk 1]
```diff
@@ -298,10 +298,16 @@ suite('Node', function() {
...
-    
+
     assert.equal(parent.firstChild, c1);
     assert.equal(c1.nextElementSibling, c2);
     assert.equal(c2.nextElementSibling, c3);
   });
 
+  test('textContent ignores comments', function() {
+    var div = document.createElement('div');
+    div.innerHTML = '<!-- comment -->';
```
Comment (arv) [inline test/js/Node.js]:
``` js
div.innerHTML = 'ab<!--cd-->ef';
assert.equal(div.textContent, 'abef');
```

seems slightly better.

Can you also add?

``` js
test('textContent of comment', function() {
  var comment = document.createComment('abc');
  assert.equal(comment.textContent, 'abc');
});
```

**Comment by arv** [comment]
LGTM

**Comment by blois** [comment]
Updated, thanks!

**Comment by arv** [comment]
LGTM. Do you mind squashing these into one commit?

**Comment by blois** [comment]
Squashed!

**Pull request merged**